src/buildercore/terraform.py

```diff
@@ -1229,6 +1229,46 @@ def _render_eks_master_security_group(context, template):
         'tags': security_group_tags,
     })
 
+def _render_eks_addon(context, template, name, addon_name, version, configuration_values=False, resolve_conflicts='OVERWRITE', service_account_role_arn=False):
+    if (version == 'latest'):
+        template.populate_data(
+            'aws_eks_addon_version',
+            'eks_addon_%s' % name,
+            block={
+                'addon_name': addon_name,
+                'kubernetes_version': '${data.aws_eks_cluster.main.version}',
+                'most_recent': True,
+            }
+        )
+
+    resource_block = {
+        'cluster_name': '${data.aws_eks_cluster.main.id}',
+        'addon_name': addon_name,
+        'addon_version': version if version != 'latest' else '${data.aws_eks_addon_version.eks_addon_%s.version}' % name,
+        'tags': aws.generic_tags(context),
+        'resolve_conflicts': resolve_conflicts,
+    }
+
+    if (configuration_values != False):
+        resource_block['configuration_values'] = configuration_values
+
+    if (service_account_role_arn != False):
+        resource_block['service_account_role_arn'] = service_account_role_arn
+
+    template.populate_resource(
+        'aws_eks_addon',
+        'eks_addon_%s' % name,
+        block=resource_block,
+    )
+
+def _render_eks_addons(context, template):
+    if (lookup(context, 'eks.addons.kube-proxy')):
```

Comment:
this is python, we don't need to wrap expressions in braces unnecessarily.

Reply:
Nothing to apologise for, it's valid syntax just not idiomatic. It even looks a little lisp-y which through me:

```clojure
(if (lookup (context "eks.addons.kube-proxy")) ; ...
```